README.md

```diff
@@ -127,9 +126,7 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] 🚧 You did a great job documenting your project! 💯 However please make sure that you have a LICENSE and that its respective link in the README is working, you can follow these steps from [GitHub's official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to set up a LICENSE. Microverse suggests using MIT! 🤓 🔧